public/lib/downloadHttpConcurrentProgress.js

```diff
@@ -260,7 +284,12 @@
         this._progressResults = {};
         this._progressCount = 0;
         this._running = true;
+        this.interval = null;
         this.start();
+        var self = this;
+        this.interval = setInterval(function () {
```

Comment:
self is not needed.
this.interval = setInterval(function () {
 this._monitor();
 }.bind(this), 100);